tests/CMakeLists.txt

```diff
@@ -1,12 +1,4 @@
-set(TESTS_SRC
-  main.cpp
-  test_config.cpp
-  test_edges.cpp
-  test_node_sets.cpp
-  test_nodes.cpp
-  test_report_reader.cpp
-  test_selection.cpp
-)
+file(GLOB TESTS_SRC "*.cpp")
```

Comment:
@mgeplf do you want also this without glob? I will close these 2 together once I get your response in any of them